index.js

```diff
@@ -67,7 +67,7 @@ function Primus(server, options) {
   // connect to the server.
   //
   this.spec = {
-    timeout: options.timeout,
+    pingInterval: options.pingInterval,
```

Comment:
I'm not against it but I don't see a real value, the changelog will specify that these options (`timeout`, `ping`, and pong) have been removed and if they are still provided there shouldn't be side effects, no?